cmd/operator-opamp-bridge/internal/config/config_test.go

```diff
@@ -212,10 +273,9 @@ func TestLoad(t *testing.T) {
 			}
 			got := NewConfig(logr.Discard())
 			err := LoadFromFile(got, tt.args.file)
-			if !tt.wantErr(t, err, fmt.Sprintf("Load(%v)", tt.args.file)) {
+			if err != nil && tt.wantErr(t, err, fmt.Sprintf("Load(%v)", tt.args.file)) {
```

Comment:
@jaronoff97 
this line didn't work before.
I understood that if err is not nil and wantErr is satisfied, the process should not continue any further.
That's why I changed it this way. Isn't that correct?

Reply:
how did it not work before? We want to check both cases here, i.e. if we expect an error (where wantErr = assert.Error) and there is no error, then we should fail. And if we don't expect an error and get one we should fail too.

Reply:
@jaronoff97
I understood that in cases where wantErr is not NoError, meaning when an error should occur, it was intended to terminate at this return statement and not execute the following code. However, in reality, even in cases where an error should occur, it wasn't returning from the if statement and was continuing to execute the next lines.
It seems to be fixed now.

Reply:
I see, regardless I think we should modify this line to serve its original purpose. I think what you could do to modify this would be change wantErr to be a boolean and then check if wantErr is true, run assert.Error(t, err)

Reply:
@jaronoff97
I fixed it.

Reply:
@jaronoff97 hello can you review this please?